### PR TITLE
Add route support to vesting feature

### PR DESCRIPTION
**Pull request by sebastinez**
This allows people to either search for their vesting contract through the search form or visit directly `vesting/<contract address>`

Was not able to test withdraw functionality, but this PR does not touch it, so should be fine.

Small example on the functionality

https://user-images.githubusercontent.com/7912302/203993327-8d2d0e8b-da1a-4001-97dc-945045cb6368.mov

### CODE REVIEW

**Comment by cloudhead** [comment]
Was going to suggest this :+1:

**Comment by rudolfs** [comment]
Looks like there are new fields: start time, cliff period end and vesting period end now. Wondering if we should show the timestamp?

Screenshot 2022-12-01 at 17 46 49

**Comment by sebastinez** [comment]
Ok wrt to some tests, I think we would need for the `JsonRpcProvider` refactor PR to happen, before this can be tested thorougly.

File: src/base/vesting/Index.svelte [hunk 1]
```diff
@@ -1,58 +1,42 @@
 <script lang="ts">
```
[the remark this answers is not in the document]
Comment (sebastinez) [reply src/base/vesting/Index.svelte]: 8b7db86f

**Comment by rudolfs** [comment]
> Or do you have any particular concerns in showing the timestamps?

Oh, it was more about the actual time part: `00:00:00 GMT`. Can we drop that?

**Comment by sebastinez** [comment]
> Oh, it was more about the actual time part: `00:00:00 GMT`. Can we drop that?

da0d980c
Bildschirm­foto 2022-12-02 um 06 48 55

**Review by rudolfs — APPROVED** [review]
🚀

**Pull request merged**